doc/angelscript/game.h

```diff
@@ -288,7 +288,7 @@ class game
 	 * @param instanceName The unique name that you chose when spawning this object
 	 * @param pos The position where the object should be moved to
 	 */
-	void moveObjectVisuals(const Ogre::String& instanceName, const Ogre::Vector3& pos);
+	void moveObjectVisuals(const string instanceName, const vector3 pos);
```

Comment:
The header is only used to generate the [AngelScript documentation](http://developer.rigsofrods.org/angelscript/)